### ISSUE TITLE
Make OmniBase and STS work with Unicode strings

### ISSUE THREAD

**Issue by brunobuzzi** [dolphinsmalltalk/Dolphin#580]
Omnibase and STS will need some work to fix them up to work with UTF
encoded strings. I'm not sure that anyone is using STS, but if you are
hopefully you will be motivated to fix it.

OmniBase and STS were removed in:
19fba6804f8aa964e15b1dee8258f89cf6a675db

**Comment by blairmcg** [comment]
I think these should be hosted externally, if anybody wants to fix them up and keep them them going. I'll be deleting them out of the main Dolphin repo at some point.

**Comment by daniels220** [comment]
To the extent anyone remembers, what are the actual issues with STS/OmniBase in Dolphin 7.1? I was able to revert the removal, boot an image, create a repository, and everything seems to work as normal. I saved a method with non-ASCII characters in the source and that worked fine, and was able to version and compare packages as well. I guess I can imagine *reading* from a repository created by an older version of Dolphin and containing methods with non-ASCII characters in the source being a problem, since they would have been in ANSI format, but that can be worked around.
